src/gameData/spriteInteractions.js

```diff
@@ -10,72 +17,105 @@ export const TRUST_LEVELS = {
   bonded: 3,
 };
 
+function sD(action) {
+  store.dispatch(action);
+}
+
 function nameOrWild(sprite) {
   if (sprite.trust > TRUST_LEVELS.friendly) {
     return sprite.name;
   }
   return `the ${sprite.species}`;
 }
 
+function chooseText(sprite, templates, trust, distance) {
```

Comment:
This logic can probably be split and moved directly into the each event itself. Feel free to add a TODO or save this for a later change, but I think we should do that.